powerhome/playbook
# [PLAY-306] Add position prop to filter kit

<img width="1789" alt="Screen Shot 2022-09-13 at 3 11 49 PM" src="https://user-images.githubusercontent.com/8194056/189978559-060b6b80-aa40-4dbc-a30b-79cbe8623eca.png">

#### Screens

<img width="1789" alt="Screen Shot 2022-09-13 at 3 11 49 PM" src="https://user-images.githubusercontent.com/8194056/189978559-060b6b80-aa40-4dbc-a30b-79cbe8623eca.png">

#### Breaking Changes

No breaking changes. Added a `placement` prop in which the user can specify the preferred position of the `popvover`. The default value now is `bottom-start`.

#### Runway Ticket URL
[
[PLAY-306] ](https://nitro.powerhrg.com/runway/backlog_items/PLAY-306)

#### How to test this

Run `./test.sh` and check for failing tests.

Try implementing the kit in multiple ways. 

Check for errors in the console.

#### Checklist:
